patches/v8/dcheck.patch

```diff
@@ -3,6 +3,7 @@ From: Ales Pergl <alpergl@microsoft.com>
 Date: Mon, 22 Oct 2018 10:47:12 -0700
 Subject: dcheck.patch
 
+https://github.com/auchenberg/volkswagen
```

Comment:
Wait what 😆

Reply:
DCHECKs failing? No problem. Here's one easy trick your compilers don't want you to know.